src/select/index.es

```diff
@@ -54,7 +54,7 @@ class Select extends InputComponent {
 
   renderOption(item, index) {
     const { key } = getOption(item);
-    return <option value={index} key={key}>{key}</option>;
+    return <option value={index} key={index + key}>{key}</option>;
```

Comment:
why is this necessary? do we have the same text option several times? how does that work?

Reply:
![image](https://user-images.githubusercontent.com/27007902/84260571-f07a3500-ab22-11ea-8b14-8347a337e510.png)

In this example we have businesses with same name(key in here) but with different IDs. 
So all is good till the moment we are rendering them as list, and they are receiving keys from the name prop. 
And passing other keys to Select won't really work since Select is using key prop from here to render options in select element, as far as it seems.